eloquent/release-build.yaml

```
@@ -0,0 +1,94 @@
+%YAML 1.1
+# ROS buildfarm release-build file
+---
+abi_incompatibility_assumed: true
+
+build_environment_variables:
+  RTI_NC_LICENSE_ACCEPTED: 'yes'
+
+jenkins_binary_job_priority: 79
+jenkins_binary_job_timeout: 120
+jenkins_source_job_priority: 75
+jenkins_source_job_timeout: 30
+notifications:
+  emails:
+  - michael+build.ros2.org@openrobotics.org
+  - ros2-buildfarm-eloquent@googlegroups.com
+  maintainers: true
+sync:
+  package_count: 1
```

There's no reason to sync until ros_workspace is built which depends on ament_package and ament_cmake_core.

```suggestion
  package_count: 3
```